### DISCUSSION TITLE
Vending machines unaffected by merchant price sliders

### DISCUSSION THREAD

**Discussion by ReesesCups** [FakeFishGames/Barotrauma#13511] [Unstable feedback]
I was testing out the abyssal difficulty which increases the price of everything across the board, and I discovered that vending machines are seemingly unaffected by the price modifiers. Evidence is attached. 
![ethanol 2](https://github.com/FakeFishGames/Barotrauma/assets/108965160/5e406dd7-f2a1-45c5-9ba2-af9752ccfb6c)
![ethanol](https://github.com/FakeFishGames/Barotrauma/assets/108965160/b4fde9db-2140-407a-8357-f7440579ef16)

**Comment by pavel4win** [comment]
still not fixed in v1.6.17.0

**Comment by Regalis11** [comment]
The description of the "shop price multiplier" says it affect purchase prices in _shops_, there's no indication that it'd affect vending machines. But I agree it would make sense for it to affect vending machines too, but imo a low-prio thing to address.

**Comment by pavel4win** [comment]
I disagree about the low priority.
We increase the price multiplier to make the game harder. 
The vending machines sell the most necessary (at the start of the game) things.
The goods in the Vending machine should cost more than the merchant sells.
And if someone can buy cheap in the Vending machine, this is a cheat and a bug.
It is probably easy to fix. Spend a minute, please.

**Comment by SomeRandomNoobKekeke** [comment]
> The goods in the Vending machine should cost more than the merchant sells.

It's debatable

In reality mb yes, and if items would cost less in vending machine then merchant could buy them and resell at a higher price

But from my expirience of playing before the sliders update i was never so short on "the most necessary things" to buy those overpriced things in vending machines, so i never touched them and this whole game mechanic was totally useless

**Comment by SomeRandomNoobKekeke** [comment]
> It is probably easy to fix

It's hacky to fix, because those are not merchants, those are fabricators and changes will affect all fabricators

But it's probably possible to multiply RequiredMoney by some neutral reputation price factor on Recipe creation

**Comment by pavel4win** [comment]
I agree that vending machines is not a very necessary mechanic. I myself have never used them. Until one day I found myself at an abandoned outpost without a drop of welding fuel. There are two working vending machines there and then I understood all their usefulness.
Also, this is the only way to get a pomegranate seed for growing right at the first outpost (if you don't steal).

In fact, do it at your own discretion. Everything I wrote is nothing more than my IMHO. I think you have to do without my advices.

Lets's add a casino. Slot machines, blackjack and of course girls.
